SConstruct

```diff
@@ -330,7 +370,7 @@ elif env["platform"] == "android":
 
     # Validate API level
     api_level = int(env["android_api_level"])
-    if env["android_arch"] in ["x86_64", "arm64v8"] and api_level < 21:
+    if "64" in env["arch"] and api_level < 21:
         print("WARN: 64-bit Android architectures require an API level of at least 21; setting android_api_level=21")
         env["android_api_level"] = "21"
         api_level = 21
```

Comment:
Is this true for host_platform windows, platform android, arch arm8 ?
I don't follow how architecture_aliases works.. if you think it's all right don't worry

Reply:
The aliases are a SCons feature. Since we have `"armv8": "arm64"`, this means that if someone writes `arch=armv8` then it will be `arm64` once we read `env["arch"]` in the script.

Note that I also manually use the aliases when automatically determining the architecture (ex: `platform.machine()` gives `AMD64` on Windows which is converted to lowercase `amd64` which is then mapped to `x86_64`).

Reply:
All right I get it, EnumVariable does the mapping so there is no change to not have that 64 appended to arch. Great.
About the manual detection, yeap I saw that, I was concerned for when arch was explicitly added and its name didn't contains 64. 
Very nice